### PR TITLE
Add option to disable management of rundir

### PR DESCRIPTION
**Pull request by tmanninger**
I am using php7.4 from sury on debian 10.
After every server reboot, this module creates the /var/run/php-fpm directory (which is not used by php-fpm) and triggers a reload, which resets all open connections to the php-fpm daemon.

This PR added an option to disable the managing of the /var/run/php-fpm directory

### CODE REVIEW

**Comment by tmanninger** [comment]
Any feedback?

**Comment by smortex** [comment]
Closing and re-openning to trigger CI. The PR is old and failing CI results are not available anymore.  @tmanninger can you check the reason for failure and fix your PR accordingly. If you need help feel free to ask for help in this PR.

File: manifests/fpm/config.pp [hunk 1]
```diff
@@ -104,14 +105,16 @@
...
-  )
+  if $manage_run_dir {
+    ensure_resource('file', '/var/run/php-fpm',
+      {
+        ensure => directory,
+        owner  => 'root',
+        group  => $root_group,
+        mode   => '0755',
+      }
+    )
```
Comment (smortex) [inline manifests/fpm/config.pp:116]: If we can skip resource management, I guess we can prefer using actual resources rather that relying on `ensure_resource()`:

```suggestion
    file { '/var/run/php-fpm':
      ensure => directory,
      owner  => 'root',
      group  => $root_group,
      mode   => '0755',
    }
```

**Review by smortex — COMMENTED** [review]
lol, so this is passing now :shrug: Just added some in-line comment for style that looks improvable but maybe here are dragons.

**Comment by tmanninger** [comment]
Can you merge die MR?

**Comment by root-expert** [comment]
@tmanninger Please rebase with our master branch since there have been 2 releases since then 😄

**Comment by tmanninger** [comment]
I created a new MR: https://github.com/voxpupuli/puppet-php/pull/674 @root-expert 
You can close this MR.

**Pull request closed without merging**